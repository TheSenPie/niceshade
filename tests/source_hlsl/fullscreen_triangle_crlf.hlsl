//T:fullscreen_triangle_crlf vs:VSMain ps:PSMain

#include "inc/triangle.hlsl"

// T: fullscreen_triangle_crlf_vertexonly vs:VSMain

float4 PSMain(Triangle_PSInput ps_in) : SV_TARGET {
	return ps_in.position;
}

Triangle_PSInput VSMain(uint vid : SV_VertexID) {
	return Triangle(vid, 1.0);
}