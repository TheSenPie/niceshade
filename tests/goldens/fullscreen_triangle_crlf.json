{
"header": {
  "magic_number": 3735928559,
  "header_size": 32,
  "version_maj": 0,
  "version_min": 1,
  "pipeline_layout_offset": 32,
  "image_to_cis_map_offset": 40,
  "sampler_to_cis_map_offset": 44,
  "user_metadata_offset": 48
},
"pipeline_layout": {
  "descriptor_sets": [
    {
      "set": 0,
      "descriptors": [
      ]
    }
  ]
},
"image_to_cis_map": {
  "entries": [
  ]
},
"sampler_to_cis_map": {
  "entries": [
  ]
},
"user_metadata": {
}
}
